openrpc.json

```diff
@@ -22,19 +22,19 @@
               "newPendingTransactions",
               "syncing"
             ],
-            "description": "The type of subscription to create. Must be one of:`newHeads`: new block headers - `logs`: logs matching a filter object - `newPendingTransactions`: new pending transactions - `syncing`: changes in syncing status"
+            "description": "The type of subscription to create. Must be one of the following: 1) `newHeads` - New block headers. 2) `logs` - Logs matching a filter object. 3) `newPendingTransactions` - New pending transactions. 4) `syncing` - Changes in syncing status."
           }
         },
         {
           "name": "filterOptions",
           "schema": {
             "title": "filterOptions",
             "type": "object",
-            "description": "An optional object containing filter options specific to the subscription type. Only applicable for 'logs' subscription type.",
+            "description": "An object containing filter options specific to the subscription type. Only applicable for the `logs` subscription type.",
```

Comment:
It looks like the whole filterOptions object is Optional since there's no required = true param? @shanejonas can you confirm if we should describe it as Optional?

Reply:
I was unsure about this because the API plugin displays this parameter as "required": https://docs.metamask.io/wallet/reference/eth_subscribe/

Is it required only for logs subscriptions?